app/home.tmpl

```diff
@@ -146,17 +150,62 @@
                     {{ $path = regexReplaceAll ".*?Path(Prefix)?\\(\\`([a-z0-9-\\/]+)\\`\\).*" $host_rule "$2" }}
                 {{end}}
 
-                {{ $entrypoint_label := print "traefik.http.routers." $service_name ".entrypoints" }}
-                {{ $entrypoint := index $container.Labels $entrypoint_label }}
-                {{ $protocol := "" }}
-                {{ if eq $entrypoint "websecure" }}
-                    {{ $protocol = "https" }}
-                {{ else if eq $entrypoint "web" }}
-                    {{ $protocol = "http" }}
-                {{ else }}
-                    {{/* entrypoint not supported */}}
+            {{ $entrypoint_label := print "traefik.http.routers." $service_name ".entrypoints" }}
+            {{ $entrypoints := index $container.Labels $entrypoint_label }}
+            {{ $protocol := "" }}
+
+            {{ $entrypoint_label := print "traefik.http.routers." $service_name ".entrypoints" }}
+            {{ $entrypoints := index $container.Labels $entrypoint_label }}
+            {{ $protocol := "" }}
+
+            {{/* Check if entrypoints label is defined and split into an array */}}
+            {{ if $entrypoints }}
```

Comment:
Can you refactor this if to something like 
```
{{ if not $entrypoints }}
  {{continue}}
{{ end }}
```
and remove the `else` block below

Reply:
Not sure what you exactly mean

Reply:
You are right, I was a bit cryptic

The flow here is written like 
```
if entrypoint
  lot of lines
else
  continue
```

Having an `else` block so far away from the `if` decrease readability

You should refactor to something like
```
if not entrypoints
  continue
end if

lot of lines
```

This way is much more readable